unity-renderer/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/LoadWrapper/LoadWrapper_GLTF.cs

```diff
@@ -33,9 +34,11 @@ public override void Load(string targetUrl, Action<LoadWrapper> OnSuccess, Actio
 
 
             if (customContentProvider == null)
-                loadHelper = new RendereableAssetLoadHelper(this.entity.scene.contentProvider, entity.scene.sceneData.baseUrlBundles);
+                loadHelper = new RendereableAssetLoadHelper(this.entity.scene.contentProvider, entity.scene.sceneData.baseUrlBundles,
```

Comment:
We shouldn't stop the client when a given asset fails importing, because this means that bad authored content can crash the client. The `GLTFShape` or `WearableController` should fail silently and continue loading.

Also, if we need some way of getting failure information, we should use `OnFail` instead of passing this new dependency around. Maybe we can change `OnFail` signature to take an `Exception` parameter if we want the details.

Reply:
In case the asset fails importing in a way that was supported, the load will continue as always. Only will report a critical error when an exception that was not being handled before occurs (an exception that used to block the loading). You can check `AssetPromise_AB.OnLoad` included `DCLCoroutineRunner.Run` which handles these exceptions that im mentioning.

I dont have a clean way for injecting the critical error reporting, since load objects needs empty constructors due inheritence (`LoadWrapper`, `LoadableShape`, etc). I could try to hook it somewhere on the standard fail stream but as far as i investigated is way too complicated, and this workaround was avoiding major refactors